lib/src/layout_adjustments/frame.dart

```diff
@@ -50,9 +68,40 @@ class Frame extends StatelessWidget {
       width: width,
       height: height,
       child: Align(
-        alignment: alignment,
+        alignment: _alignment(textDirection),
         child: child,
       ),
     );
   }
+
+  AlignmentGeometry _alignment(TextDirection direction) {
```

Comment:
It would be possible to make the `TextDirection` default to `ltr` but Flutter itself tries hard not to do that, so I'm leaving it as a required parameter: `toFlutter(TextDirection textDirection)`.